pkgs/applications/networking/circumflex/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, less, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "circumflex";
+  version = "2.6";
+
+  src = fetchFromGitHub {
+    repo = "circumflex";
+    owner = "bensadeh";
+    rev = "2.6";
+    sha256 = "sha256-pcY2PXiOazKAi8mAAbmftXDae01fcUw/u9JPOHQVclI=";
+  };
+
+  subPackage=".";
```

Comment:
Pretty sure this should be OK to remove.

Reply:
You are right! I removed it and everything still ran fine. I would assume its the default value?

Reply:
Yes.